lib/assets/chart_builder/main.js

```diff
@@ -0,0 +1,223 @@
+export function init(ctx, info) {
+  ctx.importCSS("main.css");
+  ctx.importCSS("https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap");
+
+  ctx.root.innerHTML = `
+    <div class="app">
+      <div id="info-box" class="info-box"></div>
+      <div id="data-info-box" class="info-box"></div>
+      <div>
+        <div class="row">
+          <div class="box">
+            <div class="field">
+              <label class="input-label">Chart</label>
+              <select class="input input--sm" name="chart">
+                <option value="bar">Bar</option>
+                <option value="line">Line</option>
+                <option value="area">Area</option>
+                <option value="point">Point</option>
```

Comment:
We want to order from the most common type, but I'm not sure how to establish this. @josevalim any opinion?

```suggestion
                <option value="point">Point</option>
                <option value="line">Line</option>
                <option value="bar">Bar</option>
                <option value="area">Area</option>
```

Reply:
No idea. Why not alphabetically? :)

Reply:
In this particular case alphabetically doesn't seem right, because `Area` is probably the rarest of these four? :smile:

Reply:
I feel that Bar and Line are the most common/familiar (especially for beginners) so my suggestion is Bar, Line, Point, Area. Wdy?